tools.repos

```diff
@@ -3,3 +3,7 @@ repositories:
     type: git
     url: https://github.com/autowarefoundation/autoware_tools.git
     version: main
+  tools/auto_msgs:
```

Comment:
```suggestion
  tools/external/autoware_auto_msgs:
```
Because the repository is outside the autowarefoundation org, I put it in external.
And keep the original repository name.